helper/chart_data.js

```diff
@@ -168,11 +168,27 @@ exports.generateObservationReportForNonRubricWithoutDruid = async function (data
 
   formattedCombinedAnswerArr = replaceWithLabelsOptimized(formattedCombinedAnswerArr);
 
+  for (let k = 0; k < formattedCombinedAnswerArr.length; k++) {
+    if (formattedCombinedAnswerArr[k].responseType == 'matrix') {
+      let answers = formattedCombinedAnswerArr[k].answers;
+      for (let i = 0; i < answers.length; i++) {
```

Comment:
@borkarsaish65 use meaningful variables inside for loop conditions

Reply:
done